README.md

```diff
@@ -56,16 +123,38 @@ export const fetchTodos = duck.defineAction(FETCH_TODOS, {
     };
   },
 });
+```
+
+The second method that the `duck` object provides us with is `addReducerCase`
+
+`addReducerCases(actionName: String, reducerCases: Object)`
+This method is used similarly to `defineAction` but does **not** define a new action. It is used when a different duck (to the one that creates the action) needs to take some action when an action is created. Therefore the `actionName` needs to be of an existing action and the `reducerCases` cannot have a `creator`.
```

Comment:
"take some action when an action is created" -> "change its state when an action defined in another duck is dispatched"